.gitignore

```diff
@@ -1,3 +1,3 @@
 composer.lock
 vendor
-.idea
\ No newline at end of file
+.idea
```

Comment:
Yes, I removed it first but re-added it because I wasn't sure if the maintainer of this package saw it differently.